include/os/freebsd/spl/sys/ccompile.h

```diff
@@ -43,10 +43,7 @@ extern "C" {
 #endif
 
 #define	EXPORT_SYMBOL(x)
-#define	module_param(a, b, c)
-#define	module_param_call(a, b, c, d, e)
-#define	module_param_named(a, b, c, d)
-#define	MODULE_PARM_DESC(a, b)
```

Comment:
It seems this change depends on 284580c8780e056f4f33b89f49eeeedeef9af26b, otherwise it doesn't builds on FreeBSD.